tlog.spec

```diff
@@ -136,13 +128,12 @@ rm -r %{buildroot}/usr/include/%{name}
 
 %pre
 getent group %{name} >/dev/null ||
-    groupadd -r %{name}
+    groupadd --system %{name}
 getent passwd %{name} >/dev/null ||
-    useradd -r -g %{name} -d %{_localstatedir}/run/%{name} -s /sbin/nologin \
-            -c "Tlog terminal I/O logger" %{name}
+    useradd --system --gid %{name} --home-dir %{_localstatedir}/run/%{name} \
+    --shell /sbin/nologin --comment "Tlog terminal I/O logger" %{name}
 
 %post
-/sbin/ldconfig
```

Comment:
This is still needed for older distributions (RHEL 6/7) and other distributions (Debian/Ubuntu, SUSE, etc.).